### ISSUE TITLE
New dataset

### ISSUE THREAD

**Issue by osamagharbi** [vztu/VIDEVAL_release#3]
Thank you very much for your great work, it is very helpful, I want to test the model on other datasets can you please guide me how to  to train it on my dataset please ?, thanks in advance!

**Comment by vztu** [comment]
Hi Osama, you should first 

- prepare a `csv` file storing the filename, MOS, video height, width, framerate, similar to `./features/KONVID_1K_metadata.csv`

- Then run `./demo_compute_VIDEVAL_feats.m` to extract the features of your videos (Don't forget to change the parameters [here](https://github.com/tu184044109/VIDEVAL_release/blob/0d35d6db21d4a21a536af17f3f5902ce8bc67b29/demo_compute_VIDEVAL_feats.m#L11)).

- After you have the features and MOS scores ready, run `demo_eval_BVQA_feats_one_dataset.py` and you will see the SRCC/KRCC/PLCC/RMSE results (Don't forget the change the parameters [here](https://github.com/tu184044109/VIDEVAL_release/blob/0d35d6db21d4a21a536af17f3f5902ce8bc67b29/demo_eval_BVQA_feats_one_dataset.py#L45)).

**Comment by osamagharbi** [comment]
Thank you very much for your response, it worked for me very well, except that i am getting an MSE = 3.907 between the groundtruth and the prediction which is very high in my situation. so can you please tell me how could i tune the model or retrain it again on my dataset please, thanks in advance for your time.

**Comment by vztu** [comment]
What range of MOS you used in your database? It does not make much sense to see the MSE alone, since MSE is correlated to the label range. SRCC / PLCC are more informative in this sense. What SRCC/PLCC did you get on your dataset? 

Also, VIDEVAL is intended to measure **real-world distortions** which may not necessarily perform well on single distortion dataset like only contains various levels of compression artifacts. I am not sure about your application scenario, but I recommend you to try a few more models such as NIQE, BRISQUE, and compare them.

**Comment by osamagharbi** [comment]
Thanks for your time, my dataset contains five types of real-world distortions like blur, noise and haze and I already tried NIQE and BRISQUE and now i want to use deep learning for VQA. The SRCC/PLCC i got for the predictions using your model are: PLCC=0.087, SRCC=0.15. this is why I wanted to train the model with my dataset. I will very thankful if you provide me with the code to train the model please.

**Comment by vztu** [comment]
I see. If you run `demo_eval_BVQA_feats_one_dataset.py` on your own dataset, it is just trained and tested on your own dataset.

**Comment by osamagharbi** [comment]
Thank you very much for your help, the code is working perfectly.

**Comment by vztu** [comment]
I am glad it works on your data now, good luck

**Comment by Jillian2017** [comment]
@vztu Hi, is it possible to test a dataset without  MOS values?

**Comment by vztu** [comment]
Hi @Jillian2017, yes you can run this demo and get predictions for each video: https://github.com/vztu/VIDEVAL#quality-prediction-with-pre-trained-model. Without MOS, you'd have to find out a way to evaluate the predictions.
